packages/host/app/services/matrix-service.ts

```diff
@@ -1313,7 +1313,7 @@ export default class MatrixService extends Service {
           ? JSON.parse(event.content.data)
           : event.content.data
       ) as CommandResultWithOutputContent['data'];
-      this.ensureCardFragmentsLoaded(data.cardEventId, roomData);
+      await this.ensureCardFragmentsLoaded(data.cardEventId, roomData);
```

Comment:
Sometimes there is an error says "no cardFragments found". And I noticed that we forgot to add await to this line. It caused card message event are able to be processed without card fragment event.

Reply:
nice catch, I wish there were a way to lint against this but I gather it’s not that simple